contracts/Tokens/Prime/README.md

```diff
@@ -208,14 +211,21 @@ Then we will update the `userRewardIndex` (`interests[market][account]`) to thei
 
 ## Income collection and distribution
 
-Interest reserves (part of the protocol income) from core pool markets is sent to PSR ([Protocol Share Reserve](https://github.com/VenusProtocol/protocol-reserve)). Based on configuration a certain percentage of spread income from Prime markets is reserved for prime token holders. The interest reserves will be sent to the PSR every 10 blocks (this can be changed by the community via [VIP](https://app.venus.io/governance)).
+Interest reserves (part of the protocol income) from core pool markets are sent to the PSR ([Protocol Share Reserve](https://github.com/VenusProtocol/protocol-reserve/blob/develop/contracts/ProtocolReserve/ProtocolShareReserve.sol)) contract. Based on configuration a certain percentage of spread income from Prime markets is reserved for prime token holders. The interest reserves will be sent to the PSR every 10 blocks (this can be changed by the community via [VIP](https://app.venus.io/governance)).
 
-The PSR has a function `releaseFunds` that needs to be invoked to release the funds to the Prime contract and it also has a function `getUnreleasedFunds` to get the unreleased funds for a given destination target.
+The PSR has a function `releaseFunds` that needs to be invoked to release the funds to the Prime contract, and it also has a function `getUnreleasedFunds` to get the unreleased funds for a given destination target.
 
 Prime contract takes the total released + unreleased funds and distributes to prime token holders each block. The distribution is proportional to the score of the prime token holders.
 
 When a user claims their rewards and if the contract doesn’t have enough funds then we trigger the release of funds from PSR to Prime contract in the same transaction i.e., in the `claim` function.
 
+Two key integration points in the PSR contract related with the Prime contract:
+
+* In the PSR contract, whenever a distribution configuration changes it first calls the `accrueInterest` on the Prime contract, and it releases funds to Prime (that will invoke also the `updateAssetsState` function on the Prime contract). This way, pending funds for Prime are distributed and allocated before any changes in the percentage of the protocol income for Prime.
+
+* In the PSR contract, whenever before funds are released to the Prime contract, it first calls the `accrueInterest` and then after funds are sent it calls the `updateAssetsState`.
```

Comment:
* Prior to releasing funds to the Prime contract, the PSR contract calls `accrueInterest` and then after funds are sent it calls `updateAssetsState`.

Reply:
See [c72a86](https://github.com/VenusProtocol/venus-protocol/pull/358/commits/c72a8647cd6f359fecee2616c7d45206ff9c2cb2)